File: GUIA 1/EJERCICIOS/6/ConvertirSegundos.cpp
#include <iostream>
using namespace std;

int convertirASegundos(int horas, int minutos, int segundos) {
    return horas * 3600 + minutos * 60 + segundos;
}

int main() {
    int h, m, s;
    cout << "Ingrese horas: ";
    cin >> h;
    cout << "Ingrese minutos: ";
    cin >> m;
    cout << "Ingrese segundos: ";
    cin >> s;

    int totalSegundos = convertirASegundos(h, m, s);
    cout << "Equivalente en segundos: " << totalSegundos << endl;

    return 0;
}

File: GUIA 1/EJERCICIOS/6/ConvertirSegundos.java
import java.util.Scanner;

public class ConvertirSegundos {

    // Método para convertir horas, minutos y segundos en segundos
    public static int convertirASegundos(int horas, int minutos, int segundos) {
        return horas * 3600 + minutos * 60 + segundos;
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Ingrese horas: ");
        int h = sc.nextInt();
        System.out.print("Ingrese minutos: ");
        int m = sc.nextInt();
        System.out.print("Ingrese segundos: ");
        int s = sc.nextInt();

        int totalSegundos = convertirASegundos(h, m, s);
        System.out.println("Equivalente en segundos: " + totalSegundos);

        sc.close();
    }
}
